### PR TITLE
Create vet_clinic db and a table animal

### PR DESCRIPTION
**Pull request by selma-belhadj**
# What does this PR do?
This PR is raised to create a database vet_clinic and a table animal.

## Description of the task to be completed
- Create vet_clinic database
- Create animals table
- manipulate data
## How should it be tested
- clone the repository
`git clone https://github.com/selma-belhadj/VetClinicDB.git`
- navigate to the folder
`cd VetClinicDB`
- checkout the feature
`git checkout db-structure`

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    name  varchar(100),          
+    date_of_birth  date,           
+    escape_attempts int,          
+    neutered  boolean,
+    weight_kg real
```
Comment (Dipeshtwis) [inline schema.sql]: - [x] I would recommend including the `id` column in the `animals` table.

[OPTIONAL]

- You can mark the `id` column as primary key.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @selma-belhadj, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:+1: PR looking professional.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (selma-belhadj) [reply schema.sql]: @Dipeshtwis 
It is done.

**Review by Lordkaito — APPROVED** [review]
Hi @selma-belhadj, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**